### ISSUE TITLE
loss.py only understands monai losses, but not MSELoss

### ISSUE THREAD

**Issue by markus-hinsche** [kbressem/trainlib#37]
https://github.com/kbressem/trainlib/blob/main/trainlib/loss.py

MSELoss is defined in `torch.nn`

**Comment by kbressem** [comment]
I'll rewrite the loss.py to accept more general loss functions. But until then, try patching loss.py. Loss, model, optim and transforms can all be overwritten by patch.
